scripts/setup_env_linux_debian.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+# setup dev env on debian
+
+# check if sudo permission is not available
+if [ "$EUID" -ne 0 ]; then
+    echo "sudo permission is not available"
+    exit
+fi
+
+# install system-wide dependencies 
+apt-get install python3.6 python-pip
+pip install virtualenv
+
+read -p "Python3.6 executable path: " pythonpath
```

Comment:
Question: is there a requirement for `portal` app to use a certain python 3.6 version ? If there is, I would just recommend using `pyenv` and save the day. I had pasted the instructions somewhere as I remember.